### ISSUE TITLE
Error generating code in Local with Anthropic Key

### ISSUE THREAD

**Issue by jme418** [abi/screenshot-to-code#303]
Hi there! I saw that in hosted version a message saying I need to suscribe to a plan to use an Anthropic key appeared once I dropped an screenshot so I thought I had to be suscribed to an Anthropic plan and not a screenshot-to-code one (my fault).

Now I'm trying to use the credit I have but the only thing I am getting are errors. Now I will show all I did and the error that I am getting:

## backend

**.env file**:

![image](https://github.com/abi/screenshot-to-code/assets/90529450/7d972999-7f49-4f0f-b175-ebca0c2080cb)

**other commands**:

![image](https://github.com/abi/screenshot-to-code/assets/90529450/b4a618a7-e8a2-45d2-b88c-72f338e2ff6f)

## frontend

![image](https://github.com/abi/screenshot-to-code/assets/90529450/6666bb4d-f68c-43b1-9d19-f94d997e9190)

## error

![image](https://github.com/abi/screenshot-to-code/assets/90529450/bcc33db9-2d58-4087-9b0a-55fceeffedfb)

That is all I have done I do no ¡t see any log indicating a problem or anything like that so I need help please. Thanks.

**Comment by abi** [comment]
Thanks for sharing all the screens. Seems like you followed the steps correctly. 

1. Can you visit 127.0.0.1:7001 in your browser? What does it look like?
2. Can you open your browser console after you drop in an image and let me know what's in there?

**Comment by jme418** [comment]
Thanks to you for answering! Looks like the problem has something to do with what you mentioned...

1.

![image](https://github.com/abi/screenshot-to-code/assets/90529450/bfbc626a-331b-4423-81ea-1924b22a96ef)

2.

![image](https://github.com/abi/screenshot-to-code/assets/90529450/c1c96cc1-f6fc-450e-b908-f3bc95cbda20)

**Comment by abi** [comment]
yeah, looks like your backend server is not actually running at that address. You're definitely running it locally on the same computer as you're using the browser on?

**Comment by jme418** [comment]
> yeah, looks like your backend server is not actually running at that address. You're definitely running it locally on the same computer as you're using the browser on?

Yes, 99.999% sure.

**Comment by abi** [comment]
Can you try localhost:7001 in your browser?

**Comment by abi** [comment]
Make sure it's http, not https

**Comment by jme418** [comment]
> Can you try localhost:7001 in your browser?

Of course, and it looks like I am getting the exact same error.

![image](https://github.com/abi/screenshot-to-code/assets/90529450/8386ebda-51ec-4003-b19c-0b026c5500ea)

I have tried with http, https and different browsers with cache being refreshed. Also cloning the repository once again and re-executing the commands.

**Comment by jme418** [comment]
@abi, if I do uvicorn command changing port to 8000 for example, look at what I get.

![image](https://github.com/abi/screenshot-to-code/assets/90529450/85d39b56-7d28-425b-b381-c6f91dde258e)

**Comment by abi** [comment]
Amazing. That means you can switch the front-end to connect to :8000 and everything should work. See https://github.com/abi/screenshot-to-code?tab=readme-ov-file#%EF%B8%8F-faqs "How can I update the backend host that my front-end connects to?"

If it works, please close this issue :)

**Comment by jme418** [comment]
> Amazing. That means you can switch the front-end to connect to :8000 and everything should work. See https://github.com/abi/screenshot-to-code?tab=readme-ov-file#%EF%B8%8F-faqs "How can I update the backend host that my front-end connects to?"
> 
> If it works, please close this issue :)

It works! Closing issue with a smile in my face :)
